### DISCUSSION TITLE
Option to Customize Links Styling

### DISCUSSION THREAD

**Discussion by terraHDB** [GitbookIO/community#325] [Feature Requests]
Some users of our docs have reported that links were not as visible as they could be. We'd love to be able to style all links from the Gitbook app, rather than make a thousand tiny changes to the markdown.

**Comment by widal001** [comment]
This would also be a valuable feature from an accessibility and 508 compliance standpoint. We're hoping to use GitBook for Federal open source documentation, but current 508 compliance standards require that:

> Color alone is not used to distinguish links from surrounding text unless the luminance contrast between the link and the surrounding text is at least 3:1 and an additional differentiation (e.g., it becomes underlined) is provided when the link is hovered over or receives focus.

Ideally we could style links so that they are always underlined or at least style all of them to use a text color with a higher contrast

**Comment by rodrcastro** [comment]
Hey @terraHDB and @widal001 👋 Thank you both for your suggestions! We definitely hear you. This is something our team is aware, and we'd love to offer more customization in the future. For now, we don't have plans to offer that. However, it's important to note that customizing the primary colour in a space changes the link colours.

To get more flexibility, I'd suggest going to the customization options in your space, and under the "themes" section - where you change your primary colour - you can select the "custom" option. It opens up two other colour options: header and header links. Hopefully this helps a bit for now!
